### ISSUE TITLE
add config value for --use-web-endpoint

### ISSUE THREAD

**Issue by calebdw** [mricon/b4#48]
Hello!

It would be nice to be able to set this option globally and on a per repo basis.

Thanks!

**Comment by blmaier** [comment]
Hi Caleb,

This does sound useful for people using the web endpoint. You may want to try asking on the mailing list though, the maintainer seems more active there.

Btw b4 uses git-config for all its configuration. So you can set per-repo settings using `git config` in a repo without the global option or in your `~/.gitconfig` using `includeIf ...`.

**Comment by calebdw** [comment]
> Btw b4 uses git-config for all its configuration.

Yes I'm aware, the problem is that `--use-web-endpoint` is only a cli switch and does not have a backing config value
